.github/workflows/update_metadata.yml

```diff
@@ -6,34 +6,41 @@ name: Update Metadata
 on: [workflow_dispatch]
 
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        node-version: [16.x]        
+  update-metadata:
+    runs-on: ubuntu-latest        
 
     steps:
     - name: Check Out Blockly
       uses: actions/checkout@v2
       with:
         ref: 'develop'
 
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.node-version }}        
+        node-version: 16.x
+
+    - name: Build Blockly
+      run: npm run build:compressed
+
+    - name: Build Blockly blocks
+      run: npm run build:blocks
+
+    - name: Gzip Blockly
+      run: gzip -k build/blockly_compressed.js
+
+    - name: Gzip Blockly blocks
+      run: gzip -k build/blocks_compressed.js
```

Comment:
Having these steps logged in the CI output is nice, but I think it might be better to put these in `update_metadata.sh`—both to be consistent with `check_metadata.sh`, and because it allows the script to be run locally without having to do the `gzip` manually.

Reply:
Good call, moved them to update_metadata.sh.